### PR TITLE
feature: add course content for keys,wallets and signatures

### PR DESCRIPTION
**Pull request by 0xciph3r**
### What does this PR do?
- This PR contains the curriculum outline for the btrust builder programme for the first week. It includes course resource and exercises on setting up bitcoin and lnd from source, building your first bitcoin wallet, generating keys, addresses, wallets and signatures

### How should this be tested?
- Everything here is written in markdown, previewing it in a markdown editor would be the best way to review this PR

### Any Background Context?
- It is still a work in progress, subsequent reviews and feedbacks can alter current implementation

### CODE REVIEW

File: 01-2024-schedule/week-1.md [hunk 1]
```diff
@@ -1,7 +1,112 @@
...
+  * The differences between keys, address and digital signatures
+* Gain experience with developing and using a Bitcoin wallet UI
+* Introduction to Technical Writing 
+  * Learn the critical basics of technical writing
+  * Develop skills for making documentation more accessible to all.
+
+
+## Monday & Tuesday
+
+1. Introduction to Qala
```
Comment (vladimirfomene) [inline 01-2024-schedule/week-1.md]: I think you meant to write `Introduction to Btrust Builders Programme`.

**Review by vladimirfomene — CHANGES_REQUESTED** [review]
Make the following change and I will merge it.

**Comment by vladimirfomene** [comment]
What are they going to be doing on Friday? I will added some extra materials in the evening.

**Comment by 0xciph3r** [comment]
> What are they going to be doing on Friday? I will added some extra materials in the evening.

I was thinking that friday would be a little bit less on building and more on discussing how the first week went through the feelings friday meeting. Also they could learn about wallet recovery and extra readings on multisignature wallets and how to implement it

File: 01-2024-schedule/week-1.md (hunk 1 above)
Comment (0xciph3r) [reply 01-2024-schedule/week-1.md]: I have made the necessary changes

**Review by vladimirfomene — APPROVED** [review]

**Pull request merged**